### PR TITLE
Release / ci jobs 

### PR DESCRIPTION
**Pull request by cpanato**
- Release / ci jobs

### CODE REVIEW

File: .github/workflows/ci-build-test.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v2.4.0
+
+      - uses: actions/setup-go@268d8c0ca0432bb2cf416faae41297df9d262d7f # v2.2.0
+        with:
+          go-version: '1.19'
+          check-latest: true
+          cache: true
+
+      - name: build
+        run: make test
```
Comment (06kellyjac) [inline .github/workflows/ci-build-test.yaml:43]: Do the tests actually pass currently?
In my experience https://github.com/chainguard-dev/vex/blob/main/pkg/attestation/attestation_test.go#L29 is broken

Comment (puerco) [reply .github/workflows/ci-build-test.yaml:43]: OK, I checked it and some of the tests are not up to date with the last minute changes, I'm fixing them to enable the ci jobs

Comment (cpanato) [reply .github/workflows/ci-build-test.yaml:43]: it is broken now but we are working to fix it

**Review by puerco — APPROVED** [review]

**Comment by puerco** [comment]
Lets merge it and work to unblock the tests

**Pull request merged**